packages/financial-templates-lib/test/price-feed/DefiPulseTotalPriceFeed.js

```diff
@@ -0,0 +1,151 @@
+const { DefiPulseTotalPriceFeed } = require("../../src/price-feed/DefiPulseTotalPriceFeed");
+const { NetworkerMock } = require("../../src/price-feed/NetworkerMock");
+const winston = require("winston");
+require('dotenv');
+
+contract("DefiPulseTotalPriceFeed.js", function() {
+  let defiPulseTotalPriceFeed;
+  let mockTime = 1538018200;
+  let networker;
+
+  const apiKey = process.env.DEFIPULSE_API_KEY;
+
+  const lookback = 3600*24; //???????????????
```

Comment:
what's up with this comment?

Reply:
This was a note to myself that I was unclear about an appropriate lookback time.  The API only changes at hourly intervals so the bots would run less frequently than others like cryptowatch.